src/i18n/fr-fr/index.js

```diff
@@ -0,0 +1,364 @@
+            too_small: 'La valeur minimum pour int{ size } est { min }',
+            too_large_pow2: 'La valeur maximum pour int{ size } est 2^{ size } - 1',
+            too_small_pow2: 'La valeur minimum pour int{ size } est -(2^{ size }) + 1',
+            too_large_unsigint: 'La valeur maximum pour uint{ size } est 2^{ size } - 1',
+            no_negative_unsigint: 'La valeur pour uint{ size } ne peut pas être négative',
+            invalid_signed_integer: 'Entier signé non valide',
+            invalid_unsigint: 'Entry must be a valid unsigned integer',
+            str_input_placeholder: '["une valeur", ... , "valeur final"]',
+            str_input_hint: 'Les guillemets dans les strings doivent être échappés (\\")',
+            address_placeholder: 'Adresse commençant par 0x',
+            address_label: '{ label } (adresse)',
+            boolean_array_label: '{ label } (bool[{ size }])',
+        },
+        health: {
+            status: 'Statut',
+            checked_at: 'Checked at',
+            task: 'Tâche',
+            message: 'Message',
+            block_height: 'Bloc',
+            latency: 'Latence',
+            click_to_change_format: 'Cliquer pour changer de format',
+        },
+        contract_tab: {
+            copy_abi_to_clipboard: 'Copy contract ABI JSON to clipboard',
+            enter_amount: 'Select number of decimals and enter an amount, this will be entered for you into the function parameter as uint256',
+            result: 'Resultat',
+            view_transaction: 'Voir la transaction',
+            code: 'Code',
+            read: 'Lire',
+            write: 'Ecrire',
+            amount: 'Montant',
+            value: 'Valeur',
+            custom_decimals: 'Decimales personalisés',
+            custom: 'Personalisé',
+            unverified_contract_source: 'Ce contrat n\'a pas été vérifié.',
+            click_here: 'Cliquez ici',
+            upload_source_files: 'to upload source files and verify this contract. ' +
+                'Alternatively, you can interact with the contract using an arbitrary ABI:',
+            use_erc20_abi: 'Utiliser l\'ABI ERC20',
+            use_erc721_abi: 'Utiliser l\'ABI ERC721',
+            abi_from_json: 'ABI depuis JSON',
+            paste_abi_json_here: 'Coller votre ABI JSON ici',
+            abi_json_preview: 'ABI JSON prévisualisation',
+            provided_abi_invalid: 'L\'ABI founni n\'est pas valide',
+            provided_json_invalid: 'Le JSON fourni n\'est pas valide',
+            read_functions: 'Fonctions de lecture',
+            write_functions: 'Fonctions de d\'écriture',
+            unverified_contract: 'Contract non verifié',
+        },
+        header: {
+            sign_in: 'Connexion',
+            sign_out: 'Déconnexion',
+            liq_staking: 'Staking Liquide',
+            goto_staking: 'Aller au Staking',
+            goto_health_monitor: 'go to Heath Monitor page',

```

Comment:
last one I found... for now... hahahah...

Reply:
It is still a work in progress, I haven't translated everything yet, just doing it when I got downtime